sorted-array.js

```diff
@@ -29,8 +28,13 @@ function SortedArray(values, equals, compare, getDefault) {
 SortedArray.SortedArray = SortedArray;
 
 Object.addEach(SortedArray.prototype, GenericCollection.prototype);
+GenericCollection = null;
```

Comment:
I am unclear what nulling these references achieves since they are retained by the module system.